src/core_atmosphere/dynamics/mpas_atm_time_integration.F

```diff
@@ -5136,6 +5282,8 @@ subroutine atm_compute_dyn_tend_work(nCells, nEdges, nVertices, nVertLevels_dumm
       ! Compute u (normal) velocity tendency for each edge (cell face)
       !
 
+      !$acc parallel default(present)
+      !$acc loop gang worker private(wduz, q)
```

Comment:
Optimization thing (not necessary for PR): I think everywhere that `private` is used in this port could be changed to an `acc cache` statement within the top-level of the loop.

Reply:
Let's consider the `cache` directive as part of future optimization work.

Reply:
Agreed to leave it